src/components/collaborative-editing/use-yjs/formats/collab-caret/CollabCaret.stories.tsx

```diff
@@ -38,6 +38,29 @@ function CollabCaret( { label, color } ) {
 	);
 }
 
+function CollabCaretMultiLine( { label, color } ) {
+	const { activeFormats } = applyCarets( { formats: [], text: 'foo' }, [ { label, color, start: 0, end: 0 } ] );
```

Comment:
```suggestion
	const { activeFormats } = applyCarets( { formats: [], text: 'foo' }, [ { label, color, start: 21, end: 381 } ] );
```

I just noticed we need a range here so it doesn't show a collapsed caret with no selection.

Reply:
Nice, thanks!